source/agora/common/DNS.d

```diff
@@ -616,6 +617,16 @@ public struct ResourceRecord
         return ResourceRecord(name, TYPE.A, CLASS.IN, ttl, RDATA(ipv4));
     }
 
+    /// Make a record of URI type
+    public static ResourceRecord make (TYPE type : TYPE.URI) (
+        Domain name, uint ttl, Address uri, ushort prio = 0,
+        ushort weight = ushort.max) @safe
```

Comment:
Why `max` ? `0` sounds like a better default, as not caring about the weight probably means that the entry is more important than one that sets it.

Reply:
Value towards `0` for `priority` means higher priority but value towards `max` higher priority for `weight`

Reply:
So default-created records will be at highest priority ?